README.rst

```diff
@@ -106,6 +112,28 @@ following open-access paper:
 *Note:* This paper contains out-of-date number for the size of Charliecloud’s
 code. Please instead use the current number in the FAQ.
 
+If you would like to acknowledge any of the papers in the *How do I learn more?* section, 
+you can find their citations below:
+
+* Reid Priedhorsky and Tim Randles. “Linux Containers for Fun and Profit in HPC”, 2017. In 
+  *Proc. USENIX ;login: Fall 2017, Vol. 42, No. 3*. 
+  https://www.usenix.org/system/files/login/articles/login_fall17_03_priedhorsky.pdf.
+
+* Reid Priedhorsky and Tim Randles. "Charliecloud: unprivileged containers for user-defined 
+  software stacks in HPC", 2017. In *Proc. Supercomputing*. DOI: `10.1145/3126908.3126925 <https://doi.org/10.1145/3126908.3126925>`_.
```

Comment:
I forgot to mention we use curly quotes in the source `.rst` files.

Reply:
What is this about? The DOI link reference?

Reply:
It’s the paper title. E.g. you want:

```
Reid Priedhorsky and Tim Randles. “Charliecloud: unprivileged containers for user-defined 
  software stacks in HPC”, 2017.
```

Reply:
It’s particularly subtle because Sphinx curlifies the HTML pages automatically and usually gets it right. I think maybe it’s so the man pages get curly quotes too??